### ISSUE TITLE
GC-base module should not add dropped table's contents to live-content-references.

### ISSUE THREAD

**Issue by ajantha-bhat** [projectnessie/nessie#5210]
In `IdentifyLiveContents`, we are adding the "dropped table" files to live contents when a table's dropped time is newer than cutoff time /cutoff of commit. Which leads to not cleaning up of dropped table's files. Which is not same as "dropped reference" behaviour.

It may cause confusion to the users. Better not add dropped table's contents to the live-content-references.

**Comment by snazy** [comment]
TBH, I do not understand what you mean with "dropped table" here. The code you referenced is about a `Put` operation, right?

**Comment by ajantha-bhat** [comment]
> TBH, I do not understand what you mean with "dropped table" here. The code you referenced is about a Put operation, right?

As we are filtering out only the PUT operation, if the table(key) is dropped (DELETE operation), we are ignoring that commit and still adding its contents into the live commits?

**Comment by snazy** [comment]
Yes, the behavior is intentional. Dealing w/ _completely unreferenced_  tables is a different topic. You've seen the chapter `Potential future enhancements` in #5209
